src/containers/BooksList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const mapStateToProps = state => ({ books: state.books });
+
+const BooksList = ({ books }) => (
+  <table>
+    { books.map(book => (
+      <Book book={book} key={book.id} />
+    ))}
+  </table>
+);
+
+const connectedBookList = connect(mapStateToProps)(BooksList);
```

Comment:
This can be done in line 22 like this `export default connect(mapStateToProps)(connectedBookList)`